<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FDessiner</class>
 <widget class="QMainWindow" name="FDessiner">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>305</width>
    <height>190</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dessins</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout" rowstretch="80,5,15">
    <item row="0" column="1">
     <widget class="QSlider" name="verticalSlider">
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Permet de zoomer selon l'axe Y : &lt;span style=&quot; font-weight:600; font-style:italic;&quot;&gt;operation visible lorsque le mode &amp;quot;Normal&amp;quot; est selectionne.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="0" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
     <widget class="QLabel" name="LabDessin">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <layout class="QHBoxLayout" name="HLayActions" stretch="80,20">
      <item>
       <widget class="QGroupBox" name="GrpVisu">
        <property name="title">
         <string>Mode de visualisation</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QRadioButton" name="RadCadre">
           <property name="toolTip">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Permet d'afficher l'ensemble des courbes de maniere a occuper toutes la zone graphique.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="text">
            <string>Cadre</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="RadNormal">
           <property name="toolTip">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Permet d'afficher l'ensemble des courbes en conservant leurs dimensions initiales.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="text">
            <string>Normal</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="RadAucun">
           <property name="text">
            <string>Aucun</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="ChkEffacer">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Peremt de valider ou devalider l'affacement de la zone graphique lors de chaque action de dessin.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>Effacer</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QSlider" name="horizontalSlider">
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Permet de zoomer selon l'axe X : &lt;span style=&quot; font-weight:600; font-style:italic;&quot;&gt;operation visible lorsque le mode &amp;quot;Normal&amp;quot; est selectionne.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(255, 0, 255);</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
